.github/sync.yml

```diff
@@ -47,7 +48,7 @@ group:
         dest: .github/CODEOWNERS
   # Automattic repositories
   - repos: |
-      Automattic/openverse-infrastructure
+      WordPress/openverse-infrastructure
```

Comment:
Looks like this can be part of the Backend repositories! 😃

Reply:
👍 Yes, we can 🗑️ the entire `# Automattic repositories` section.

Reply:
On it!